assets/i18n/hi.json

```diff
@@ -0,0 +1,267 @@
+    "dgc-windows-wallet-modal-title": "आपका Wallet चुनें।",
+    "dgc-windows-wallet-modal-multidoge": "MultiDoge",
+    "dgc-windows-wallet-modal-multidoge-desc": "Dogecoin लाइट Wallet.",
+    "dgc-windows-wallet-modal-dogecoin-core-32bit": "Dogecoin कोर। (32 बिट)",
+    "dgc-windows-wallet-modal-dogecoin-core-64bit": "Dogecoin कोर। (62 बिट)",
+    "dgc-windows-wallet-modal-dogecoin-core-desc": " Dogecoin \"Full\" Wallet.",
+    "dgc-windows-wallet-modal-dogecoin-core-guide": "समुदाय-नियत उपयोगकर्ता नियमावली",
+    "dgc-windows-wallet-modal-getting-started": "जानें कि कैसे के साथ अपने wallet स्थापित करने के लिए<a href=\"/getting-started\" target=\"_blank\">गेटिंग स्टार्टेड गाइड</a>.",
+    "dgc-windows-wallet-modal-differences": "दोनों में क्या अंतर हैं?",
+    "dgc-windows-wallet-modal-sync": "Dogecoin का उपयोग करने के लिए आपको ब्लॉकचेन के साथ सिंक करना होगा।",
+    "dgc-windows-wallet-modal-ledger": "इस का मतलब है, आप Dogecoin की जरूरत \"ledger\" to use Dogecoin.",
+    "dgc-windows-wallet-modal-light-wallet": "Multidoge एक \"Light\" wallet है। यह \"Skimming\" द्वारा blockchain साथ समन्वयित हो जाता blockchain के माध्यम से, तेजी से सिंक बार प्रदान करते हैं।",
+    "dgc-windows-wallet-modal-full-wallet": "दूसरी ओर, डॉगकोइन कोर एक \"Full\" wallet है। यह इसे डाउनलोड करने, एक ठोस काम कर Dogecoin बटुआ प्रदान करके समन्वयन करती है।",
+    "dgc-windows-wallet-modal-prosandcons": "फायदे और नुकसान?",
+    "dgc-windows-wallet-modal-different-strokes": "दोनों wallets के अपने फायदे और नुकसान हैं।",
+    "dgc-windows-wallet-modal-multidoge-pro": "MultiDoge एक हल्का wallet है, इसलिए इसे प्रयोग करने योग्य होने से पहले केवल blockchain के माध्यम से \"Skim\" करना होगा। यह जल्दी से किया जाता है।",
+    "dgc-windows-wallet-modal-multidoge-con": "लेकिन बटुआ केवल सामान्य उपयोग के लिए होना चाहिए। यह <strong> नहीं कर सकते </ strong>, मेरा Dogecoin लिए इस्तेमाल किया जा के रूप में बटुआ धीमी और अनुत्तरदायी हो जाएगा अगर इस तरह से इस्तेमाल किया।",
+    "dgc-windows-wallet-modal-dogecoin-core-con": "दूसरी ओर Dogecoin कोर, पूरे blockchain डाउनलोड करता है। इसकी प्रारंभिक समन्वयन काफी धीमी MultiDoge की तुलना में है, और यह एक बहुत अधिक स्थान लेता है।",
+    "dgc-windows-wallet-modal-dogecoin-core-pro": "लेकिन यह एक लाभ में आता है। यह इस बटुए के साथ Dogecoin मेरी करने के लिए, के रूप में यह भुगतान से लेनदेन को संभालने के लिए सक्षम हो जाएगा सुरक्षित है। यह भी मदद Dogecoin नेटवर्क को बनाए रखेगा।",
+    "dgc-osx-wallet-modal-dogecoin-core": "Dogecoin कोर।",
+	"dgc-android-wallet-modal-title": "Google Play से डाउनलोड करें।",
+    "dgc-android-wallet-modal-apk": "<a href=\"https://github.com/langerhans/dogecoin-wallet-new/releases/download/1.08/wallet-1.08-aligned.apk\">प्रत्यक्ष APK डाउनलोड।</a>",
+	"dgc-ios-wallet-modal-title": "App स्टोर से डाउनलोड करें।",
```

Comment:
Since you left google play in english completely it's advisable to keep app store in english completely too

Reply:
Done